docs/en/executing_sql_queries/bind-parameters.md

```diff
@@ -0,0 +1,139 @@
+# Binding parameters
+
+When creating a **DB** command from a **SQL with parameters**, you should almost always use the approach of binding parameters to prevent **SQL injection attacks**. You can **bind parameters** to a **SQL statement** by using named placeholders or question mark placeholders. Named placeholders are of the form `:name` and question mark placeholders are of the form `?`. The following example shows how to **bind parameters** to a **SQL statement**:
```

Comment:
```suggestion
When creating a **DB** command from a **SQL with parameters**, you should usually use the approach of binding parameters to prevent **SQL injection attacks**. You can **bind parameters** to a **SQL statement** by using named placeholders or question mark placeholders. Named placeholders are of the form `:name` and question mark placeholders are of the form `?`. The following example shows how to **bind parameters** to a **SQL statement**:
```

Reply:
There's no example after `:`.